### DISCUSSION TITLE
DAG中的Node添加的input组件无法输入

### DISCUSSION THREAD

**Discussion by qqabcv520** [antvis/XFlow#91] [Q&A] [answered]
使用DAG例子，在ReactNode中添加的input组件无法输入，光标一闪而过
复现地址
https://codesandbox.io/s/zi-ding-yi-lian-xian-luo-ji-forked-4dzsl?file=/react-node/algo-node.tsx

**Comment by lyn-boyu** [comment, marked as answer]
https://github.com/antvis/X6/pull/1754 solved in this issue. please upgrade X6 version
